packages/svelte/tests/runtime-runes/samples/snippet-expressions/child.svelte

```diff
@@ -0,0 +1,7 @@
+<script>
+	let { snippets, snippet, optional } = $props();
+</script>
+
+{@render snippets[snippet]()}
```

Comment:
It'd be good to have some additional cases here; weird stuff like `snippets?.[0]?.()` -- if more-arbitrary patterns like that aren't supported then this could get pretty far into the unfathomable uncanny valley ("Why can I do `foo.bar[baz]()` and `foo.bar?.()` but not `foo?.bar?.[baz]?.()`?")